.github/workflows/update-fink-version-file.yml

```diff
@@ -0,0 +1,49 @@
+name: Update Fink Version File
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "inlang/source-code/editor/*"
+      
+jobs:
+  update-fink-version:
```

Comment:
please dont make this an extra ci step, this could be just part of the normal fink build.